src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMoviesQuery = session.createQuery("from Movie", Movie.class);
+            List<Movie> movies = getAllMoviesQuery.getResultList();
+            return movies;
```

Comment:
I think @sarakhmen meant that this should be fixed in this way: 
```suggestion
           return session.createQuery("from Movie", Movie.class).getResultList();
```
Check please other similar cases in alls dao classes.